commands/src/main/java/org/wildfly/extras/creaper/commands/socketbindings/AddSocketBinding.java

```diff
@@ -0,0 +1,174 @@
+package org.wildfly.extras.creaper.commands.socketbindings;
+
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * <p>A command for creating socket binding under default socket binding group if not specified else.</p>
+ * <p>This command <b>does not check</b> whether the port or multicast port are available!</p>
+ *
+ * @author Jan Kasik <jkasik@redhat.com>
```

Comment:
Please remove the `@author` tag, this information is available from Git history.